apps/forward.cpp

```diff
@@ -102,3 +103,9 @@ void getHelp(char** argv)
 
     exit(0);
 }
+
+void dispEllapsed(const std::chrono::duration<double> elapsed_seconds){
+  std::cout <<  "-------------------------------------------" << std::endl;
+  std::cout <<  "| Elapsed Time: " << elapsed_seconds.count() << " s." << std::endl;
+  std::cout <<  "-------------------------------------------" << std::endl;
+}
```

Comment:
can you move this function in om_utils.h to avoid the code dupe?

Reply:
That's one option, its only half of the apps in this PR use `om_utils.h`. +1 to refactor the whole ncourses and unify all the apps. It's just that I'm not sure if it should be done here.

Reply:
And if we do it. Instead of `dispEllapsed`, we should provide iostring operator. So that we can write `std::cout << end_time-start_time;` (I think)